packages/celotool/src/e2e-tests/transfer_tests.ts

```diff
@@ -829,4 +860,77 @@ describe('Transfer tests', function(this: any) {
       })
     }
   })
+
+  describe('Transfers Frozen >', () => {
```

Comment:
Some other ideas for tests:
- Sender/receiver removed from whitelists are blocked again
- *unfreeze* allows all parties to once again transfer (regardless of whitelist state)

Reply:
just added those suggestions. thanks!